CHANGELOG.md

```diff
@@ -4,16 +4,35 @@
 
 ### `Added`
 
+- Additional output from `CometAdapter` (generated with the parameter `--pin_out`)
```

Comment:
Attached Issues are missing here?

Reply:
Based on an internal evaluation we discussed that this might be of interest to add to the comet outcome. Here all of the peptide hits are present and can be used to obtain the PSMs from the identified peptides.
There is indeed no issue, we can still generate that though (same as for all of the other points).